packages/frontend-api/src/Model/Resolver/Payment/PaymentResolver.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Payment;
+
+use Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface;
+use Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface;
+use Overblog\GraphQLBundle\Error\UserError;
+use Ramsey\Uuid\Uuid;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Payment\Payment;
+use Shopsys\FrameworkBundle\Model\Payment\PaymentFacade;
+
+class PaymentResolver implements ResolverInterface, AliasedInterface
```

Comment:
I am thinking about use case for this resolver on frontend (same as TransportResolver) :thinking:

Reply:
Well I think it might be useful in some cases

Reply:
Yes, but I can't do the case use to find :smile: